src/extension.ts

```diff
@@ -16,14 +17,25 @@ class Extension {
   constructor(private readonly context: ExtensionContext) {}
 
   registerCommands(): void {
-    const visjsPanelView = new VisjsPanelView(this.context);
-    const debuggerPanel = new DebuggerPanel(this.context, visjsPanelView);
+    const debuggerPanel = new DebuggerPanel(this.context, this.getPanelViewByConfiguration());
     const debugEventManager = new DebugEventManager();
     debugEventManager.registerDebuggerPanel(debuggerPanel);
 
     this.context.subscriptions.push(commands.registerCommand('visual-oo-debugger.openDebugger', () => debuggerPanel.openPanel()));
     this.context.subscriptions.push(commands.registerCommand('visual-oo-debugger.exportPNG', () => debuggerPanel.exportPanel()));
     this.context.subscriptions.push(commands.registerCommand('visual-oo-debugger.startGIF', () => debuggerPanel.startRecordingPanel()));
     this.context.subscriptions.push(commands.registerCommand('visual-oo-debugger.stopGIF', () => debuggerPanel.stopRecordingPanel()));
+
+    workspace.onDidChangeConfiguration(() => debuggerPanel.setPanelViewProxy(this.getPanelViewByConfiguration()));
```

Comment:
So this will reset the PanelView everytime ANY configuration is changed, right? The PanelView should only be reassigned if the value changed. Otherwise you will lose the changelog etc.

Reply:
Done as suggested -> resolved